### PR TITLE
BDKD-391 Making language consistent

### PR DESCRIPTION
**Pull request by lolo-fernandez-sirca**

### CODE REVIEW

File: clusterous/cluster.py [hunk 1]
```diff
@@ -1453,7 +1453,7 @@ def sync_get(self, local_path, remote_path):
         stdin, stdout, stderr = ssh.exec_command(cmd)
         output_content = stdout.read()
         if 'cannot access' in stderr.read():
-            message = "Error: Folder '{0}' does not exists.".format(remote_path)
+            message = "Error: Folder '{0}' does not exist.".format(remote_path)
```
Comment (balram-ramanathan-sirca) [inline clusterous/cluster.py]: As per the list, could you also remove the "Error:" prefix from the error messages?

**Comment by balram-ramanathan-sirca** [comment]
Need to remove the "Error:" prefix from some of the messages, otherwise looks good.

File: clusterous/cluster.py (hunk 1 above)
Comment (lolo-fernandez-sirca) [inline clusterous/cluster.py]: @balram-ramanathan-sirca Good point. Fixed.

Comment (balram-ramanathan-sirca) [inline clusterous/cluster.py]: @lolo-fernandez-sirca The prefix appears in a few more places as well :-)

Comment (lolo-fernandez-sirca) [inline clusterous/cluster.py]: @balram-ramanathan-sirca I see. Thanks for noticing. Fixed all now.

File: clusterous/cluster.py [hunk 2]
```diff
@@ -1483,7 +1483,7 @@ def ls(self, remote_path):
         stdin, stdout, stderr = ssh.exec_command(cmd)
         output_content = stdout.read()
         if 'cannot access' in stderr.read():
-            message = "Error: Folder '{0}' does not exists.".format(remote_path)
+            message = "Folder '{0}' does not exists.".format(remote_path)
```
Comment (balram-ramanathan-sirca) [inline clusterous/cluster.py]: Just realised that these and others should be "..does not exist" not "does not exists." (also note the lack of full stop)

Comment (lolo-fernandez-sirca) [inline clusterous/cluster.py]: @balram-ramanathan-sirca Sure. Done.

**Comment by balram-ramanathan-sirca** [comment]
Looks good, merging

**Pull request merged**